### ISSUE TITLE
how to pass null Ptr

### ISSUE THREAD

**Issue by extrawurst** [twistedfall/opencv-rust#213]
I have a stupid question:

how do I pass the C's `null` into a rust wrapper that takes `Ptr`?

example: 
```
let edge_detect =
        ximgproc::create_structured_edge_detection("model.yml", ...).unwrap();
```

the std lib `null()` seems not to be compatible.

**Comment by twistedfall** [comment]
You should be able to use `from_raw` function, something like this:
```rust
    let ptr_null = unsafe { Ptr::from_raw(std::ptr::null_mut()) };
    ximgproc::create_structured_edge_detection("model.yml", ptr_null);
```

**Comment by extrawurst** [comment]
the key missing element for this to work is:
```
use opencv::prelude::Boxed;
```

but anyhow this seg faults even though `NULL` is supposed to be a valid param here :(

**Comment by twistedfall** [comment]
I wonder whether the segfault is happening because of the bindings or OpenCV. Do you have more details?

**Comment by twistedfall** [comment]
If you could also provide the minimal reproducible example I could also check it out.

**Comment by sumibi-yakitori** [comment]
I am having the exact same problem. It occurs with the following code.
The execution environment is MacOS and OpenCV is installed from homebrew.

opencv: stable 4.6.0 (bottled)

```rust
let ptr_null = unsafe { Ptr::from_raw(core::ptr::null_mut()) };
let path = Path::new("model.yml");
if path.exists() {
  let edge_detector = ximgproc::create_structured_edge_detection(&path.to_string_lossy(), ptr_null)?;
}
```

**Comment by twistedfall** [comment]
Let me check this out. Can you also share the corresponding `model.yml`?

**Comment by sumibi-yakitori** [comment]
@twistedfall 
Thanks for the quick reply!
https://github.com/opencv/opencv_extra/blob/4.x/testdata/cv/ximgproc/model.yml.gz
I am using this file. Of course I unzip and use it.
(Incidentally, it did not make sense to make the path to the model file an absolute path)

However, it seems that the same SIGSEGV occurs when I specify the path to a model file that does not exist.

```
(signal: 11, SIGSEGV: invalid memory reference)
```

**Comment by sumibi-yakitori** [comment]
I tried to run that code in Manjaro, but got the same error.

**Comment by sumibi-yakitori** [comment]
This C++ code worked fine on my MacOS, so maybe it is the bindings issue.

```cpp
#include <opencv2/opencv.hpp>
#include <opencv2/ximgproc.hpp>

using namespace cv;
using namespace cv::ximgproc;

int main() {
  auto pDetector = createStructuredEdgeDetection("model.yml");

  return 0;
}
```

**Comment by twistedfall** [comment]
Fixed in v0.68.0, you can now pass `None` to that function
